.nvmrc

```diff
@@ -1 +1 @@
-16.16.0
+18.13.0
```

Comment:
Have you tested this branch with Alberto or Gwen (which both still use Node v16)? Will it cause version conflicts with their dependencies?